apparmor.d/profiles-a-f/firewalld

```diff
@@ -50,10 +52,11 @@ profile firewalld @{exec_path} {
   @{bin}/false                    rix,
   @{bin}/ipset                    rix,
   @{bin}/kmod                     rPx,
+  @{bin}/modprobe                 rPx,
   @{bin}/xtables-legacy-multi     rix,
   @{bin}/xtables-nft-multi        rix,
 
-  /usr/local/lib/python3.10/dist-packages/ r,
+  /usr/local/lib/python*/dist-packages/ r,
```

Comment:
```suggestion
  /usr/local/lib/python3.@{int}/dist-packages/ r,
```

Reply:
@roddhjav, ok, I checked and found `python3.@{int}` in other profiles.
At the time of the commit, I only found `python*` in openbox and speedtest profiles. They probably need to be changed.

Reply:
Maybe use `@{int2}` instead?